regular/irc.html

```diff
@@ -4,6 +4,7 @@
 
 <div class="noticebox">
   <h2>Want to join HackSoc? A good place to start is joining the chat!</h2>
+  <p>Before joining, please read our <a href="/coc.html">Code of Conduct</a>.</p>
```

Comment:
```suggestion
  <p>Before joining, please read our <a href="/coc.html">Code of Conduct</a>. If you want to know more about how we use our chat platforms, read through the Conventions on the same page.</p>
```
cc @FromAnkyra - is this what you meant?

Reply:
yup, that looks good - I'll use similar phrasing in the announcements and when I link to the page in the topic

Reply:
maybe s//Conventions/Conventions and Guidelines/, just to be consistent with the headings (sorry, this was because I said conventions)